tests/PostalNotificationChannelTest.php

```diff
@@ -46,7 +46,13 @@ public function testGetRecipients()
 
         $notifiableMock->expects($this->exactly(count($mockResponses)))
             ->method('routeNotificationFor')
-            ->with(PostalNotificationChannel::class, $notificationMock)
+            ->with(
+                $this->logicalOr(
```

Comment:
I really wanted to use `withConsecutive` as the order of calls is predictable but it did not seem to work as expected so had to use this `logicalOr` instead which I feel is less precise.